include/game_overlay.h

```diff
@@ -13,4 +13,28 @@ void Overlay_UnloadByID(const FSOverlayID param0);
 int Overlay_GetLoadDestination(const FSOverlayID param0);
 BOOL Overlay_LoadByID(const FSOverlayID param0, int param1);
 
+// TODO: this should integrate into build system instead of a manual define here.
```

Comment:
See: https://mesonbuild.com/howtox.html#set-extra-compiler-and-linker-flags-from-the-outside-when-eg-building-distro-packages